cli.py

```diff
@@ -590,10 +592,26 @@ def run_ratdecoders(file_path, passed_reporter):
         if key not in SUPER_LIST:
             others[key] = output[key]
     reporter.add_metadata("other", others)
-
     return {script_name: reporter.metadata}
 
 
+def run_mwcfg(file_path, reporter):
+    process = subprocess.run(['mwcfg', '--input', f'{file_path}', '-m', './modules'], capture_output=True)
+    output = ast.literal_eval(process.stdout.decode())
+    configs = output[0]['configs'][0]
+    extracted = {k: v for k, v in configs.items()}
```

Comment:
Why is configs being copied? It doesn't look like extracted is being modified in the loop below. Could the loop run directly on configs?

Reply:
👍 changed